pkg/nirmata/api_utils.go

```diff
@@ -41,6 +41,16 @@ func waitForClusterState(apiClient client.Client, maxTime time.Duration, cluster
 	return state.(string), nil
 }
 
+func waitForClusterDeletedState(apiClient client.Client, maxTime time.Duration, clusterID client.ID) (string, error) {
+	states := []interface{}{"deleted"}
+	state, err := apiClient.WaitForStates(clusterID, "state", states, maxTime, "")
+	if err != nil {
```

Comment:
Should we check here if the error is 404 not found and return without showing an error to the user?

Reply:
sure